addons/hide-flyout/addon.json

```
@@ -108,6 +108,12 @@
         }
       ],
       "default": "default"
+    },
+    {
+      "name": "Lock by default",
```

The category click setting doesn't have a lock.
```suggestion
      "name": "Open by default",
```